docs/eigenda/operator-guides/troubleshooting.md

```diff
@@ -76,4 +76,12 @@ again.
 
 #### What does the error "EIP1271 .. signature not from signer" mean?
 
-This indicates you have not imported your BLS key correctly. Please reconfirm the keys you imported to ensure there were no typos or mistakes.
\ No newline at end of file
+This indicates you have not imported your BLS key correctly. Please reconfirm the keys you imported to ensure there were no typos or mistakes.
+
+#### Error message "failed to update operator's socket .. execution reverted"
+
+"msg="failed to update operator's socket" !BADKEY="execution reverted: RegistryCoordinator.updateSocket: operator is not registered"
```

Comment:
this could also mean operator is not registered right? this doesn't mean rpc is at fault

Reply:
[This Discord user](https://discord.com/channels/1089434273720832071/1229391211035365496/1231524602937868308) reported the root cause of the error was a non-responsive RPC endpoint and that moving to a working RPC endpoint fixed the issue. If the operator was not registered, I believe there is a different error message?

Either way, I'm open to whichever outcome you'd like - modify or remove this item?

Reply:
Hi @shrimalmadhur can I get your preference on how you'd like to resolve this comment?

Reply:
this could be either RPC or operator not registered as well. Both are valid scenario while updating socket

Reply:
10-4 will add "operator not registered"